src/components/banner/banner.css

```diff
@@ -0,0 +1,24 @@
+.LeftColumn
+{
+    
+    max-width: 791px;

```

Comment:
use rems instead of px

Reply:
Okay

Reply:
Updated the file with px values to rem values